examples/basics/data_row_metadata.ipynb

```diff
@@ -3,7 +3,9 @@
   {
```

Comment:
<p>similar to my comment above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>